src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private static final int DIVIDER = 2;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / DIVIDER;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Shape:\t" + FigureName.RIGHT_TRIANGLE
+                + ",\tColor:\t" + this.getColor()
+                + ",\tArea:\t" + this.getArea()
+                + ",\tFirst Leg:\t" + this.getSecondLeg()
```

Comment:
There's a mistake in the `draw()` method. The `getFirstLeg()` method should be called instead of `getSecondLeg()` and vice versa to match the variable names correctly.